scripts/jenkins-yoctobuild-build.sh

```diff
@@ -28,7 +99,6 @@ else
   /bin/rm -f build/conf/local.conf
   echo $?
   /bin/cp meta-mender-qemu/build-conf/bblayers.conf  build/conf/bblayers.conf
-  sed -i -- "s,/home/jenkins/workspace/yoctobuild,$WORKSPACE,g" build/conf/bblayers.conf
```

Comment:
I would leave this. Even if it replaces it with the same value, it's better to keep it in case `WORKSPACE` changes later (and I can see that happening, if two branches become incompatible for example).